### ISSUE TITLE
Add a way to set default monster values

### ISSUE THREAD

**Issue by soul4soul** [soul4soul/ot-monster-converter#20]
Often times information is missing from creature files. The conversion process defaults missing values to their system type. So bool flags like walks around fire is false. Support can be added to allow setting specific default values for certain flags.

**Comment by EPuncker** [comment]
I strongly suggest to keep IgnoreInvisible default to true, its less abusable if it pass unnoticed, most of monsters with this value as unknow are bosses and that would be a problem 😄

**Comment by soul4soul** [comment]
> 
> 
> I strongly suggest to keep IgnoreInvisible default to true, its less abusable if it pass unnoticed, most of monsters with this value as unknow are bosses and that would be a problem 😄

Good Idea, While this issue is still open and I don't plan on fixing it for a while I changed the default value for the IgnoreInvisible  to true.

**Comment by soul4soul** [comment]
I was thinking about this issue the other day, a simple way to go about implementing this would be to have the program default to reading a local config file which provides keyed values. Possibly a basic ini style file.
